### DISCUSSION TITLE
google speech api

### DISCUSSION THREAD

**Discussion by multumescmult44** [Eloston/ungoogled-chromium#1692] [Q&A]
Hello.. lately i began to use duolingo.com and edx.org to learn italian.. my default browser firefox does not support speech recognition at both sites.. besides edx.org has many other features that work only on chrome.. i did not want to use chrome even just for these two sites.. so i tried to use ungoogled chromium.. as whole point was to cut ties with google and since, as far as i understood, these two websites use google servers to recognize and validate the speech, i cannot use these features of these websites.. i thought, instead of using chrome itself to access these features, i may have an exemption in ungoogled chromoum to connect to google speech recognition servers (if they are called so).. i may be totally wrong while thinking about these ideas.. :) would someone please help me to achieve my intention to use speech validation services of duolingo.com and edx.org via ungoogled chromium?

**Comment by wchen342** [comment]
It is currently not possible to do this in browser. If you know how to compile chromium then you can try excluding the google speech sites from domain substitution list.

**Reply by multumescmult44** [reply]
thank you very much for your answer and time.. :)
